### PR TITLE
ptg ui look and enhancements

### PR DESCRIPTION
**Pull request by poojacsawai**

### CODE REVIEW

File: apps/ptg-angular-app/src/app/dashboard/multistep-form/multistep-form-layout/multistep-form-layout.component.ts
Comment (shivrjsharma) [inline apps/ptg-angular-app/src/app/dashboard/multistep-form/multistep-form-layout/multistep-form-layout.component.ts:1]: image

Please check function naming convention

Comment (shivrjsharma) [reply apps/ptg-angular-app/src/app/dashboard/multistep-form/multistep-form-layout/multistep-form-layout.component.ts:1]: image
Please remove unused code

File: libs/ptg-ui-angular-lib/sidenav/sidenav/sidenav.component.scss
Comment (shivrjsharma) [inline libs/ptg-ui-angular-lib/sidenav/sidenav/sidenav.component.scss:1]: image

File: libs/ptg-ui-angular-lib/styles/index.scss
Comment (shivrjsharma) [inline libs/ptg-ui-angular-lib/styles/index.scss:1]: image

Please remove unused code

File: libs/ptg-ui-web-components/src/components/ptg-modal/ptg-modal.scss
Comment (shivrjsharma) [inline libs/ptg-ui-web-components/src/components/ptg-modal/ptg-modal.scss:1]: image

File: apps/ptg-angular-app/src/app/common/common.theme.scss [hunk 1]
```diff
@@ -11,24 +11,23 @@
...
         &:not(.parent-item):not(.disabled) {
-          // background-image: url("assets/Path_bg.svg");
-          // background-size: 100% 100%;
-          background-color: $side-menu-text-color;
+
+          background-color: rgb(237, 231, 246) !important;
           border-end-end-radius: 5rem;
           border-start-end-radius: 5rem;
-          color: $white;
+          color: rgb(103, 58, 183) !important;
```
Comment (ZalawaHarsha) [inline apps/ptg-angular-app/src/app/common/common.theme.scss]: why we are using rgb instead scss variables ?

File: apps/ptg-angular-app/src/app/common/common.theme.scss [hunk 2]
```diff
@@ -37,8 +36,12 @@
...
       }
   
-      // typography
-      // font-size: $sidenav-font-size;
-      font-size: 1.125rem;
+      margin: 0px;
+      letter-spacing: 0em;
+      font-weight: 400;
+      line-height: 1.5em;
+      color: rgb(54, 65, 82);
```
Comment (ZalawaHarsha) [inline apps/ptg-angular-app/src/app/common/common.theme.scss]: please use scss variables instead hardcoded color.

**Review by ZalawaHarsha — CHANGES_REQUESTED** [review]

File: apps/ptg-angular-app/src/app/common/sidenav/sidenav.component.scss
Comment (shivrjsharma) [inline apps/ptg-angular-app/src/app/common/sidenav/sidenav.component.scss:1]: image

Please check this

File: apps/ptg-angular-app/src/app/common/userprofile/userprofile.component.html
Comment (shivrjsharma) [inline apps/ptg-angular-app/src/app/common/userprofile/userprofile.component.html:1]: image

Please don't use blank class

File: apps/ptg-angular-app/src/app/dashboard/accordian-example/accordian-example.component.html
Comment (shivrjsharma) [inline apps/ptg-angular-app/src/app/dashboard/accordian-example/accordian-example.component.html:1]: image
Can we use loader instead loading... text

File: apps/ptg-angular-app/src/app/dashboard/calendarexample/calendarexample1/calendarexample1.component.html
Comment (shivrjsharma) [inline apps/ptg-angular-app/src/app/dashboard/calendarexample/calendarexample1/calendarexample1.component.html:1]: image
please remove unused code

**Review by nimishjain77 — APPROVED** [review]

File: apps/ptg-angular-app/src/app/dashboard/multistep-form/multistep-form-layout/multistep-form-layout.component.ts
Comment (poojacsawai) [reply apps/ptg-angular-app/src/app/dashboard/multistep-form/multistep-form-layout/multistep-form-layout.component.ts:1]: resolved

File: libs/ptg-ui-angular-lib/sidenav/sidenav/sidenav.component.scss
Comment (poojacsawai) [reply libs/ptg-ui-angular-lib/sidenav/sidenav/sidenav.component.scss:1]: resolved

File: libs/ptg-ui-angular-lib/styles/index.scss
Comment (poojacsawai) [reply libs/ptg-ui-angular-lib/styles/index.scss:1]: resolved

File: libs/ptg-ui-web-components/src/components/ptg-modal/ptg-modal.scss
Comment (poojacsawai) [reply libs/ptg-ui-web-components/src/components/ptg-modal/ptg-modal.scss:1]: resolved

File: apps/ptg-angular-app/src/app/common/common.theme.scss (hunk 1 above)
Comment (poojacsawai) [reply apps/ptg-angular-app/src/app/common/common.theme.scss]: resolved

File: apps/ptg-angular-app/src/app/common/common.theme.scss (hunk 2 above)
Comment (poojacsawai) [reply apps/ptg-angular-app/src/app/common/common.theme.scss]: resolved

File: apps/ptg-angular-app/src/app/common/sidenav/sidenav.component.scss
Comment (poojacsawai) [reply apps/ptg-angular-app/src/app/common/sidenav/sidenav.component.scss:1]: resolved

File: apps/ptg-angular-app/src/app/common/userprofile/userprofile.component.html
Comment (poojacsawai) [reply apps/ptg-angular-app/src/app/common/userprofile/userprofile.component.html:1]: resolved

File: apps/ptg-angular-app/src/app/dashboard/accordian-example/accordian-example.component.html
Comment (poojacsawai) [reply apps/ptg-angular-app/src/app/dashboard/accordian-example/accordian-example.component.html:1]: resolved

File: apps/ptg-angular-app/src/app/dashboard/calendarexample/calendarexample1/calendarexample1.component.html
Comment (poojacsawai) [reply apps/ptg-angular-app/src/app/dashboard/calendarexample/calendarexample1/calendarexample1.component.html:1]: resolved

**Pull request merged**